packages/graphile-build-pg/src/QueryBuilder.ts

```diff
@@ -1,32 +1,39 @@
-// @flow
 import * as sql from "pg-sql2";
-import type { SQL } from "pg-sql2";
-import isSafeInteger from "lodash/isSafeInteger";
-import chunk from "lodash/chunk";
-import type { PgClass, PgType } from "./plugins/PgIntrospectionPlugin";
+import isSafeInteger = require("lodash/isSafeInteger");
+import chunk = require("lodash/chunk");
+import { PgClass, PgType } from "./plugins/PgIntrospectionPlugin";
+import { GraphileResolverContext } from "graphile-build";
+
+export { GraphileResolverContext };
+
+type SQL = import("pg-sql2").SQL;
+export { sql, SQL };
 
 // eslint-disable-next-line flowtype/no-weak-types
```

Comment:
You can remove this `eslint-disable` comment, and all similar comments related to Flow, right?

Reply:
Yeah; in fact that was why CI was failing :+1: